app/src/androidTest/java/com/ncnf/main/MainActivityTest.java

```diff
@@ -35,48 +17,36 @@
     @Rule
     public RuleChain testRule = RuleChain.outerRule(hiltRule).around(activityTestRule);
 
-    @Before
-    public void setup(){
-        Intents.init();
-    }
-
-    @After
-    public void cleanup(){
-        Intents.release();
-    }
-  
-    @Test
-    public void test_event1() {
-        onView(withId(R.id.eventButton)).perform(click());
-        Intents.intended(hasComponent(EventActivity.class.getName()));
-    }
-
-    @Test
-    public void test_event2() {
-        onView(withId(R.id.event2Button)).perform(click());
-        Intents.intended(hasComponent(EventActivity.class.getName()));
-    }
-
-    @Test
-    public void testMapButton(){
-        onView(ViewMatchers.withId(R.id.to_map_link)).perform(click());
-        Intents.intended(hasComponent(MapActivity.class.getName()));
-    }
-
-    @Test
-    public void feedActivityButtonWorks(){
-        onView(withId(R.id.feedViewButton)).perform(click());
-        Intents.intended(hasComponent(FeedActivity.class.getName()));
-    }
-
-    @Test
-    public void profileTest(){
-        onView(withId(R.id.mainProfileButton)).perform(click());
-        if(FirebaseAuth.getInstance().getCurrentUser() != null) {
-            Intents.intended(hasComponent(UserProfileActivity.class.getName()));
-        } else {
-            Intents.intended(hasComponent(LoginActivity.class.getName()));
-        }
-    }
+//    @Before
+//    public void setup(){
+//        Intents.init();
+//    }
+//
+//    @After
+//    public void cleanup(){
+//        Intents.release();
+//    }
+
+//    @Test
+//    public void testMapButton(){
+//        onView(ViewMatchers.withId(R.id.to_map_link)).perform(click());
+//        Intents.intended(hasComponent(MapActivity.class.getName()));
+//    }
+//
+//    @Test
+//    public void feedActivityButtonWorks(){
+//        onView(withId(R.id.feedViewButton)).perform(click());
+//        Intents.intended(hasComponent(FeedActivity.class.getName()));
+//    }
+//
+//    @Test
+//    public void profileTest(){
+//        onView(withId(R.id.mainProfileButton)).perform(click());
+//        if(FirebaseAuth.getInstance().getCurrentUser() != null) {
+//            Intents.intended(hasComponent(UserProfileActivity.class.getName()));
+//        } else {
+//            Intents.intended(hasComponent(LoginActivity.class.getName()));
+//        }
+//    }
```

Comment:
Is it normal that this is all commented out?

Reply:
Yeah I wanted to test it on Cirrus before I delete everything. But it's the next thing I do

Reply:
Fair enough